packages/utxo-lib/src/coinselect/utils.ts

```diff
@@ -116,11 +116,19 @@ export function dustThreshold(feeRate: number, options: CoinSelectOptions) {
     return Math.max(options.dustThreshold, price);
 }
 
-export function uintOrNaN(v: number) {
-    if (typeof v !== 'number') return;
-    if (Number.isNaN(v) || !Number.isFinite(v) || !Number.isInteger(v) || v < 0) return;
-    return v;
-}
+export const formatStringFeeRate = (feeRate: string): number | undefined => {
```

Comment:
```suggestion
export const formatStringFeeRate = (feeRate: string) => {
```
nit: you dont have to define return type, typescript will do it for you

Reply:
Sure, it's just another guard, in my view makes sense for utils to have types defined as thoroughly as possible, so that TS throws if a return doesn't match the type after some edits :)

Reply:
right, but if you mess something here and for example return a string, then TS will catch it anyway in a place where you are using this utility (compose/index)

Reply:
but it doesn't matter really, it's only my personal preference, i would not define it.
i'm leaving this decision to you

Reply:
Ok, removed affdf675ab7412f0fd13ceaa5ec53ce84fcd708c